### PR TITLE
Add minikube example

### PR DESCRIPTION
**Pull request by tgraf**
Fixes: #775
Fixes: #657

### CODE REVIEW

File: Documentation/gettingstarted.rst [hunk 1]
```diff
@@ -9,6 +9,242 @@ your laptop.  It is designed to take 15-30 minutes.
...
+
+Step 1: Deploy Cilium
+^^^^^^^^^^^^^^^^^^^^^
+
+The next step is to deploy Cilium to your Kubernetes cluster in the form of a
+DaemonSet_ which will deploy one Cilium pod per cluster node in the
+``kube-system`` namespace along with all other system relevant daemons and
+services.
+
+To deploy Cilium, run ``kubectl create -f`` and pass in the file ``cilium-ds.yaml``
```
Comment (rlguarino) [inline Documentation/gettingstarted.rst]: I've tried to do this but I'm getting a API version error:

```
ross@alnilam:~/workspace/cilium$ kubectl create -f examples/minikube/cilium-ds.yaml
serviceaccount "cilium" created
daemonset "cilium-consul" created
daemonset "cilium" created
Error from server (BadRequest): error when creating "examples/minikube/cilium-ds.yaml": the API version in the data (rbac.authorization.k8s.io/v1beta1) does not match the expected API version (rbac.authorization.k8s.io/v1alpha1)
Error from server (BadRequest): error when creating "examples/minikube/cilium-ds.yaml": the API version in the data (rbac.authorization.k8s.io/v1beta1) does not match the expected API version (rbac.authorization.k8s.io/v1alpha1)
ross@alnilam:~/workspace/cilium$ kubectl version 
Client Version: version.Info{Major:"1", Minor:"6", GitVersion:"v1.6.3", GitCommit:"0480917b552be33e2dba47386e51decb1a211df6", GitTreeState:"clean", BuildDate:"2017-05-10T15:48:59Z", GoVersion:"go1.7.5", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"6", GitVersion:"v1.6.0", GitCommit:"fff5156092b56e6bd60fff75aad4dc9de6b6ef37", GitTreeState:"clean", BuildDate:"2017-05-09T23:19:49Z", GoVersion:"go1.7.1", Compiler:"gc", Platform:"linux/amd64"}
ross@alnilam:~/workspace/cilium$ minikube version 
minikube version: v0.19.0
ross@alnilam:~/workspace/cilium$ git show --oneline
0dfdfc9 Documentation: Provide getting started guide with minikube
```

Comment (tgraf) [reply Documentation/gettingstarted.rst]: What version of Kubernetes are you using? 1.6?

Comment (rlguarino) [reply Documentation/gettingstarted.rst]: The server is 1.6.0 (I didn't choose that minikube somehow choose that)
The client is 1.6.3

Comment (rlguarino) [reply Documentation/gettingstarted.rst]: I started minikube like this:
```
$ minikube start --network-plugin=cni --iso-url https://github.com/cilium/minikube-iso/raw/master/minikube.iso
Starting local Kubernetes v1.6.0 cluster...
Starting VM...
SSH-ing files into VM...
Setting up certs...
Starting cluster components...
Connecting to cluster...
Setting up kubeconfig...
Kubectl is now configured to use the cluster.

```

Comment (tgraf) [reply Documentation/gettingstarted.rst]: Is it minikube 0.19?

Comment (tgraf) [reply Documentation/gettingstarted.rst]: The issue seems to be that the `localkube` as part of your minikube version is still on `v1alpha1` APIs whereas the RBAC roles in the yaml are:
```
apiVersion: rbac.authorization.k8s.io/v1beta1
```
I think you can just change this to 
```
apiVersion: rbac.authorization.k8s.io/v1alpha1
```

I think we'll have to provide a guide without RBAC or be very clear about the exact version of minikube to run.

Comment (rlguarino) [reply Documentation/gettingstarted.rst]: Interesting... what is the version of `localkube` controlled by? I would have expected  the 
 version of minikube would have determined the version of `localkube` the cluster starts with, is that not the case?

Comment (tgraf) [reply Documentation/gettingstarted.rst]: Yes, the minikube version should dictate the localkube version.

**Pull request merged**